RELEASE_NOTES.md

```diff
@@ -16,27 +16,55 @@
 
 * **IMPORTANT** Systems with NVIDIA CPUs and GPUs must see [Known issues](#known-issues)
 
+### New software support
```

Comment:
Can we add the following under line 13 for "miscellaneous functionality"?

* The [System Admin Toolkit (SAT)](glossary.md#system-admin-toolkit-sat) is fully included in CSM. There is no longer a separate SAT product stream to install. SAT 2.6 releases, which accompanied CSM 1.5, are the last releases of SAT as a separate product. For more information, see [SAT in CSM](/operations/system_admin_toolkit/about_sat/SAT_in_CSM.md).

Reply:
Done